src/pages/map.html

```diff
@@ -1,6 +1,36 @@
 <div class="">
     <div class="h-28 -mt-28" id="curve-to-start-Qualitat-section" style="background: url(./assets/BGs/qualitat.svg); background-position: top; background-size: cover;"></div>
-    <div class="text-white bg-[#001342]" id="rwanda-map-section-description">Rwanda map section description</div>
+    <div class="text-white bg-[#001342] flex flex-col gap-5 px-24">
```

Comment:
You need to remove this arbitrary value. If you can't find this color you can define it in the tailwind.config.js file

Reply:
done